com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/UnlitPass.hlsl

```diff
@@ -1,13 +1,46 @@
-﻿PackedVaryings vert(Attributes input)
+﻿
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl"
```

Comment:
Currently "UniversalFragmentUnlit()" resides in Lighting.hlsl (as do all of the color calculation methods for the fragments), this is because they call various methods to calculate whether we override the color with a debug one and all of those methods are currently in here too. I'll see if I can move those methods into "Debugging3D.hlsl" instead then we wont need this dependency.

Reply:
This relies on us being able to split out all of the debugging code from "Lighting.hlsl" - currently we can do this because it relies on certain lighting methods (for access to lights, shadow-cascades and BRDF data). We would need to refactor this code in order to do this - maybe split out the light code into one file, shadow code into another and BRDF out into a third. @arttu-peltonen